app/views/measures/grouped/_table.html.erb

```diff
@@ -2,6 +2,8 @@
   <%= local_assigns[:caption] %>
 </h3>
 
+<%= yield %>
```

Comment:
```suggestion
<%= render 'measures/grouped/tariff_duty_calculator_link', declarable_code: declarable.code if local_assigns[:duty_calculator_link] %>
```

Reply:
Not sure why we need a different pattern that what we've got currently here

Reply:
changed, as discussed we choose to use local_assigns[:...]  style for consistency.